api/CSSRule.json

```
@@ -67,7 +67,8 @@
               "version_added": "4"
             },
             "ie": {
-              "version_added": null
+              "version_added": null,
```

Your note implies this feature has some support. We should mark this as `true`:

```suggestion
              "version_added": true,
```